### ISSUE TITLE
The return type of the <Transition> element doesn't appear to be valid JSX

### ISSUE THREAD

**Issue by joshuafcole** [ryansolid/solid-transition-group#1]
```
src/Schematic/Schematic.tsx(143,10): error TS2786: 'Transition' cannot be used as a JSX component. 
    Its return type '(() => unknown)[]' is not a valid JSX element. 
        Type '(() => unknown)[]' is not assignable to type 'ArrayElement'.
```

It looks like `ArrayElement` doesn't allow `FunctionElement` children in `solid-js/types/rendering/jsx.d.ts`. Does it seem reasonable to add that?

**Comment by ryansolid** [comment]
It should. It probably just doesn't like that it's unknown instead of JSX element being returned. Let me take a look. I've made some updates in recent Solid.

**Comment by ryansolid** [comment]
Ok I fixed the types in v0.0.2. Let me know it works for you.

**Comment by joshuafcole** [comment]
Working perfectly, thanks for the crazy fast turnaround time!
